charts/k8s-monitoring/tests/platform/eks-addon-with-terraform/.gitignore

```diff
@@ -0,0 +1,10 @@
+.norender
```

Comment:
did you mean to include `.norender` in the gitignore? This is a `.no-render` file but I think you want to keep that, iiuc

Reply:
That can be removed. I'll do that.